ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/InMobiAppInstallNativeAdMapper.java

```diff
@@ -128,6 +129,7 @@ void mapAppInstallAd(final Context context) {
     placeHolderView.setLayoutParams(
         new RelativeLayout.LayoutParams(
             RelativeLayout.LayoutParams.MATCH_PARENT, RelativeLayout.LayoutParams.MATCH_PARENT));
+    placeHolderView.setGravity(Gravity.CENTER);
```

Comment:
Should we not be using something like this, for example -
```
View positiveButton = findViewById(R.id.positiveButton);
RelativeLayout.LayoutParams layoutParams = 
    (RelativeLayout.LayoutParams)positiveButton.getLayoutParams();
layoutParams.addRule(RelativeLayout.CENTER_IN_PARENT, RelativeLayout.TRUE);
positiveButton.setLayoutParams(layoutParams);
```

Reply:
It doesn't matter actually as this placeholder is going to contain only one view which is primaryView. 
Also, problem with the implementation you are suggesting is that the primary view returned by the SDK has layout params set of type ViewGroup.LayoutParams so either we will have to generate a new set of RelativeLayout params and assign it on the primary view with the rule mentioned in the snippet above.